core/services/ardupilot_manager/frontend/index.html

```diff
@@ -0,0 +1,212 @@
+                    </div>
+                    <div class="cell-1">
+                        <input type="checkbox" data-role="checkbox" :checked="endpoint.protected" class="d-flex flex-justify-center" disabled />
+                    </div>
+                    <div class="cell-1">
+                        <button class="button primary alert cycle" @click="deleteEndpoint(endpoint)">
+                          <span class="mif-minus"></span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+
+            <div id="infobox" class="info-box" data-role="infobox">
+                <span class="button square closer"></span>
+                <div class="info-box-content">
+                    <h3>{{ infoboxTitle }}</h3>
+                    <p>{{ infoboxMessage }}</p>
+                </div>
+            </div>
+
+        </div>
+    </div>
+</body>
+
+<script>
+    /*global Metro axios, Vue*/
+
+    new Vue({
+      el: '#app',
+      data () {
+        return {
+          new_endpoint: {
+            name: '',
+            owner: '',
+            connection_type: '',
+            place: '',
+            argument: null,
+            persistent: false,
+            protected: false,
+          },
+          registeredEndpoints: [],
+          Metro: null,
+          infoboxTitle: '',
+          infoboxMessage: '',
+        }
+      },
+      created () {
+        this.API_URL = '/v1.0/endpoints'
+        this.connection_type_options = {
+          serial: 'Serial',
+          udpin: 'UDP Server',
+          udpout: 'UDP Client',
+          tcpin: 'TCP Server',
+          tcpout: 'TCP Client',
+        }
+      },
+      mounted () {
+        Metro.init()
+        this.Metro = Metro
+        this.fetchRegisteredEndpoints()
+        setInterval(this.fetchRegisteredEndpoints, 5000)
+      },
+      methods: {
+        fetchRegisteredEndpoints () {
+          axios({
+            method: 'get',
+            url: this.API_URL,
+            timeout: 500,
+          })
+            .then((response) => {
+              this.registeredEndpoints = response.data
+            })
+            .catch(() => {
+              this.infoboxTitle = 'Error'
+              this.infoboxMessage = 'Cannot fetch points. Check you network connection.'
+              Metro.infobox.open('#infobox')
+            })
+        },
+        createEndpoint () {
+          axios({
+            method: 'post',
+            url: this.API_URL,
+            timeout: 500,
+            data: [this.new_endpoint],
+          })
+            .then(() => {
+              this.fetchRegisteredEndpoints()
+            })
+            .catch(() => {
+              this.infoboxTitle = 'Error'
+              this.infoboxMessage = 'Could not create endpoint. Check if you are filling all fields correctly.'
```

Comment:
Can we inform what is wrong ?

Reply:
We can, but it is hard, because the http error can come on different structures depending if it was a validation error or a logical one (duplicated, for example).

I will put more time on that to come with a detailed error.